image_geometry/src/pinhole_camera_model.cpp

```diff
@@ -1,5 +1,5 @@
 #include "image_geometry/pinhole_camera_model.h"
-#include <sensor_msgs/distortion_models.h>
+//#include <sensor_msgs/distortion_models.h>
```

Comment:
Is it possible to [port this file](https://github.com/ros/common_msgs/blob/jade-devel/sensor_msgs/include/sensor_msgs/distortion_models.h) rather than replacing everything with string litterals? it looks like low effort and would avoid undoing it in the future